packages/admin/src/components/DownloadCsv.jsx

```diff
@@ -0,0 +1,426 @@
+import React, { useState, useEffect } from "react";
+import {
+  Modal,
+  Box,
+  Typography,
+  Button,
+  TextField,
+  Autocomplete,
+  IconButton,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+  FormControl,
+  FormLabel,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import DownloadIcon from "@mui/icons-material/Download";
+import { CSVLink } from "react-csv";
+import {
+  fetchStates,
+  fetchDistricts,
+  fetchBlocks,
+  fetchSchools,
+  fetchClasses,
+} from "../api/filterStudentDetails";
+
+const DownloadCsv = ({ open, handleClose, rowData }) => {
+  const [dropdownValues, setDropdownValues] = useState({
+    dropdown1: null,
```

Comment:
@pmulik34 Names should be logically given. It should give clear idea on what that is that used for.
dropdown1 could be named as stateDropdown or maybe stateList etc

Reply:
Hi @ankush-maherwal I have made those changes, you can check now